### PR TITLE
Update Questionnaire exacerbation_diary/exacerbation_diary_armt.json

### PR DESCRIPTION
**Pull request by yatharthranjan**
- confirm the type of medication

### CODE REVIEW

**Comment by afolarin** [comment]
`"field_name": "are_you_taking_any_steroid"` should probably be `are_you_taking_any_meds` 

I also think if the meds are only being taken for exacerbation (prob need to check) then we could also branch the meds question on the exacerbation question (I'm thinking of keeping this therefore to a minimum).


But otherwise LGTM

File: questionnaires/exacerbation_diary/exacerbation_diary_armt.json [hunk 1]
```diff
@@ -65,5 +65,35 @@
...
+                "code": "1",
+                "label": "Antibiotics"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[are_you_taking_any_steroid]=1",
```
Comment (afolarin) [inline questionnaires/exacerbation_diary/exacerbation_diary_armt.json:90]: `are_you_taking_any_meds]`

**Comment by yatharthranjan** [comment]
>I also think if the meds are only being taken for exacerbation (prob need to check) then we could also branch the meds question on the exacerbation question (I'm thinking of keeping this therefore to a minimum).

how would we branch off that? like if exacerbation value is low, we go to the medication questions?

**Comment by afolarin** [comment]
Yeah, I was thinking a non-zero response to the first question could be used as a branch? But perhaps we should discuss with Malik?

**Comment by yatharthranjan** [comment]
Yes because if they are taking medication, the value will be low (even be 0)?

**Comment by afolarin** [comment]
Actually you make a fair point, if treatment leads to low symptoms of exacerbation then we'd not capture that. OK on that basis I think maybe to branch off the question 1. Keep them as two questions.

**Pull request merged**